.github/workflows/static-assets-check.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         python-version: [ '3.8', '3.11' ]
         node-version: [ 18 ]
         npm-version: [ 10.5.x ]
-        mongo-version: ["4.4", "7.0"]
+        mongo-version: ["7.0"]
```

Comment:
Maybe with this cherry-pick https://github.com/openedx/edx-platform/commit/4687dae487efb072d5f1f17134c2670b4c92f62f
we could also drop support of python3.8